### ISSUE TITLE
Initial ContextMenu cells are not accessibility buttons

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/GitHawk#1948]
![GitHawk Upload by BasThomas](https://i.imgur.com/w6vq86W.jpg)

This will eg. just say “Lock” and will not announce this is a button.

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.22.0 (1532006598)
Device: iPhone X (iOS 11.4.1)
TestFlight: true
</pre>
</details>

**Comment by telip007** [comment]
Hey, I'm relatively new to the app and the project... How can I access this menu and maybe do you know in which file this is?

**Comment by telip007** [comment]
Nvm, found it :)
